stacs/vegetation-phenology-and-productivity/VPP_2022_S2_T40KCC-010m_V105_s2_TPROD/VPP_2022_S2_T40KCC-010m_V105_s2_TPROD.json

```diff
@@ -0,0 +1,238 @@
+    "http://example.com/eea/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/projection/v1.1.0/schema.json",
+    "https://stac-extensions.github.io/raster/v1.1.0/schema.json"
+  ],
+  "type": "Feature",
+  "id": "VPP_2022_S2_T40KCC-010m_V105_s2_TPROD",
+  "bbox": [55.077444, -20.877262, 56.13278, -19.8938],
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [55.077444, -20.877262],
+        [56.13278, -20.877262],
+        [56.13278, -19.8938],
+        [55.077444, -19.8938],
+        [55.077444, -20.877262]
+      ]
+    ]
+  },
+  "properties": {
+    "description": "The Total Productivity (TPROD), one of the Vegetation Phenology and Productivity (VPP) parameters, is a product of the pan-European High Resolution Vegetation Phenology and Productivity (HR-VPP) component of the Copernicus Land Monitoring Service (CLMS). The Total Productivity (TPROD), or large integral, is the growing season integral computed as the sum of all daily Plant Phenology Index values between the dates of the season start (SOSD) and end (EOSD). The Plant Phenology Index (PPI) is a physically based vegetation index, developed for improving the monitoring of the vegetation growth cycle. The PPI index values, with 5-day satellite revisit cycle, are first used in a function fitting to derive the PPI Seasonal Trajectories, which is a filtered time series with regular 10-day time step. From these Seasonal Trajectories, a suite of 13 Vegetation Phenology and Productivity (VPP) parameters are then computed and provided, for up to two seasons each year. The Total Productivity is one of the 13 parameters. The full list is available in the table 3 of the Product User Manual https://land.copernicus.eu/user-corner/technical-library/product-user-manual-of-seasonal-trajectories/ A complementary quality indicator (QFLAG) provides a confidence level, that is described in table 4 of the same manual. The TPROD dataset is made available as raster files with 10 x 10m resolution, in UTM/WGS84 projection corresponding to the Sentinel-2 tiling grid, for those tiles that cover the EEA38 countries and the United Kingdom and for two seasons in each year from 2017 onwards. It is updated in the first quarter of each year.",
+    "datetime": null,
+    "start_datetime": "2022-01-01T00:00:00.000Z",
+    "end_datetime": "2022-12-31T00:00:00.000Z",
+    "created": "2023-04-08T01:47:59.000Z",
+    "providers": [
+      {
+        "name": "Copernicus Land Monitoring Service",
+        "description": "The Copernicus Land Monitoring Service provides geographical information on land cover and its changes, land use, ground motions, vegetation state, water cycle and Earth's surface energy variables to a broad range of users in Europe and across the World in the field of environmental terrestrial applications.",
+        "roles": ["licensor", "producer", "processor", "host"],
+        "url": "https://land.copernicus.eu"
+      }
+    ],
+    "proj:epsg": 32740,
```

Comment:
curious about this. Is it ok that the proj is different from the Polygon coordinates?

Reply:
This is resulted from the definition. The `geometry` and `bbox` are [specified in WGS84](https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#item-fields). The proj:epsg under the properties is the metadata of the item and should be the actual projection. See the [Sentinel-2 example](https://github.com/stactools-packages/sentinel2/blob/main/examples/sentinel2-l2a-example/S2A_MSIL2A_20190212T192651_R013_T07HFE_20201007T160857/S2A_MSIL2A_20190212T192651_R013_T07HFE_20201007T160857.json#L26).